tests/test_introduction_to_classes.py

```diff
@@ -0,0 +1,197 @@
+import pytest
+
+from codecademy.introduction_to_classes import Animal
+from codecademy.introduction_to_classes import Customer
+from codecademy.introduction_to_classes import Fruit
+from codecademy.introduction_to_classes import ReturningCustomer
+from codecademy.introduction_to_classes import Triangle
+
+test_name = 'Apple'
+test_color = 'Green'
+test_flavor = 'Sweet'
+test_poisonous = False
+test_weight = 200
+
+
+def test_Fruit_description():
+    apple = Fruit(test_name, test_color, test_flavor, test_poisonous, test_weight)
+
+    assert apple.name == test_name
+    assert apple.color == test_color
+    assert apple.flavor == test_flavor
+    assert apple.poisonous == test_poisonous
+    assert apple.weight == test_weight
+
+    assert apple.description() == "I'm a %s %s and I taste %s." % (test_color, test_name, test_flavor)
+
+
+@pytest.mark.parametrize("bite_size, expected", [(1, 199), (100, 100), (200, 0), (300, 0)])
+def test_Fruit_bite_it(bite_size, expected):
+    apple = Fruit(test_name, test_color, test_flavor, test_poisonous, test_weight)
+    apple.bite_it(bite_size)
+    assert apple.weight == expected
+
+
+@pytest.mark.parametrize("input_neg", [
+    None,
+    True,
+    False,
+    "",
+    "A"
```

Comment:
Still need negative tests with correct input type (int/float) and invalid values: -1, -2.3, etc.